CHANGELOG.md

```diff
@@ -28,12 +28,16 @@ Current
 
 ### Fixed:
 
+- [Fix and refactor role based filter to allow CORS](https://github.com/yahoo/fili/pull/99)
+    * Fix `RoleBasedAuthFilter` to bypass `OPTIONS` request for CORS
```

Comment:
I think changelog is not the place to explain things, google CORS will give plenty of answers. What do you think? @cdeszaq